.github/workflows/main.yml

```
@@ -0,0 +1,71 @@
+name: "CI"
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+
+env:
+  CI: true
+
+jobs:
+  Test:
+    if: "!contains(github.event.head_commit.message, '[skip ci]')"
+    strategy:
+      matrix:
+        # os: [ubuntu-latest, macos-latest, windows-latest]
+        os: [macos-latest, windows-latest]
```

```suggestion
        os: [ubuntu-latest, macos-latest, windows-latest]
```
this will allow testing on linux